server/data/data-memory/src/main/java/org/apache/james/mailrepository/memory/MemoryMailRepositoryStore.java

```diff
@@ -88,13 +83,30 @@ public void init() throws Exception {
         }
     }
 
+    private void initEntry(MailRepositoryStoreConfiguration.Item item) throws ConfigurationException {
```

Comment:
then why do you have a private method here?

Reply:
I don't understand why wouldn't it be private...

Reply:
because of " - Group public methods together"

Reply:
Ah

I valued grouping `init concerns first`, then the @Overideded public method then the private ones.

Overwise I felt it would make the init() method harder to read.

Do you want me to group it with other methods in another PR?

Reply:
no that's ok for me, thx